client/src/app/components/login/login.component.ts

```diff
@@ -40,10 +40,16 @@ export class LoginComponent implements OnInit {
 
     this.authService.login(this.body).subscribe(
       (data) => {
-        this.tokenService.SetToken(data.token); // Setting token in cookie for logged in users
+        this.tokenService.SetToken(data.token); // Setting token in cookie for logged in users   
+        if(data.role === 'SA') {
+          setTimeout(() => {
+            this.router.navigate(['manage-organizations']);
+          }, 1000);
+        }else {
         setTimeout(() => {
           this.router.navigate(['']); // Redirect user to component in path:home (defined in alta-home-routing.module.ts)
         }, 1000); // Redirect the user after 1 seconds ( in case we want to add a loading bar when we click on button )
+      }
```

Comment:
This line isn’t properly indented, or is it just me?

Reply:
It's you.